HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -1200,6 +1200,74 @@ sponsor.hmcl=Hello Minecraft! Launcher 是一個免費、自由、開源的 Mine
 system.architecture=架構
 system.operating_system=作業系統
 
+terracotta=多人聯機
+terracotta.easytier=關於 EasyTier
+terracotta.terracotta=Terracotta | 陶瓦聯機
+terracotta.status=聯機大廳
+terracotta.back=退出
+terracotta.network_warning=多人連線基於 p2p，最終線上體驗和您的網路情況有較大關係。
+terracotta.sudo_installing=HMCL 需要驗證您的密碼才能安裝線上核心
+terracotta.from_local.title=線上核心第三方下載管道
+terracotta.from_local.desc=在部分地區，內建的預設下載管道可能不穩定或連線緩慢
+terracotta.from_local.guide=您應下載名為 %s 的線上核心套件。下載完成後，請將檔案拖曳到目前介面來安裝。
+terracotta.from_local.file_name_mismatch=您應該下載名為 %1$s 的線上核心包，而非 %2$s
+terracotta.status.bootstrap=正在收集信息
+terracotta.status.uninitialized.not_exist=未下載聯機核心
+terracotta.status.uninitialized.not_exist.title=下載聯機核心（約 8MiB）
+terracotta.status.uninitialized.update=需更新聯機核心
+terracotta.status.uninitialized.update.title=更新聯機核心（約 8MiB）
+terracotta.status.uninitialized.desc=您承諾，在多人聯機全過程中，您將嚴格遵守您所在國家或地區的全部法律法規
+terracotta.confirm.title=用戶須知
```

目前 HMCL 繁中其他地方都使用「使用者」一詞。

```suggestion
terracotta.confirm.title=使用者須知
```